lib/screens/about/about.dart

```diff
@@ -0,0 +1,143 @@
+import 'dart:convert';
+import 'package:flutter/gestures.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:fogosmobile/actions/contributors_actions.dart';
+import 'package:fogosmobile/models/app_state.dart';
+import 'package:fogosmobile/models/contributor.dart';
+import 'package:fogosmobile/screens/about/contributor_item.dart';
+import 'package:fogosmobile/screens/components/fire_gradient_app_bar.dart';
+import 'package:fogosmobile/utils/uri_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:redux/redux.dart';
+
+
+class About extends StatefulWidget {
+  @override
+  _AboutState createState() => _AboutState();
+}
+
+class _AboutState extends State<About> {
+
+  List<Contributor> contributors = new List();
+
+  Widget contributorsWidget(List<Contributor> contributors) {
+    return new StoreConnector<AppState, AppState>(
+        converter: (Store<AppState> store) => store.state,
+        onInit: (store) {
+          store.dispatch(LoadContributorsAction());
+        },
+        builder: (BuildContext context, AppState state) {
+          if (state.hasContributors == true){
```

Comment:
Remove the `== true`.

Reply:
== true ensures nullability check.

Reply:
Right. Ideally, this should never return a `null` but a default value (probably `false`).

Reply:
true, but i am getting them null sometimes, i might have made some mistake, but i don't find it.
Any idea why i might be getting it null?.

Reply:
Replace [this line](https://github.com/FogosPT/fogosmobile/blob/a79ee86d564c0462b06ea210f0a4d32e2cbfa8da/lib/reducers/app_reducer.dart#L81) with:
```
hasContributors ?? false
```

Reply:
Add the null aware operator to [this line](https://github.com/FogosPT/fogosmobile/blob/a79ee86d564c0462b06ea210f0a4d32e2cbfa8da/lib/reducers/app_reducer.dart#L81):
```
hasContributors ?? false
```

Reply:
that, resolved it, thanks.